docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_an_expectation_suite_with_the_user_configurable_profiler.rst

```diff
@@ -242,9 +242,369 @@ This guide will help you create a new Expectation Suite by profiling your data w
+                  metric_name: column.mean
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - parameter_name: my_parameter_min
+                  class_name: MetricParameterBuilder
+                  metric_name: column.min
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - parameter_name: my_parameter_false_positive_threshold_min
+                  class_name: MultiBatchBootstrappedMetricDistributionParameterBuilder
+                  batch_request:  # this should be unioned with the rest of the batch request from the call to profile
+                    partition_request:
+                      partition_index: "-10:"
+                  metric_configuration:
+                    metric_name: column.min
+                    metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                  p_values: # TBD Not yet implemented
+                    min_value: $variables.false_positive_threshold
+                    max_value: 1.0
+                - parameter_name: my_parameter_quantile_ranges
+                  class_name: MultiBatchBootstrappedMetricDistributionParameterBuilder
+                  batch_request:
+                    partition_request:
+                      partition_index: "-10:"
+                  metric_configuration:
+                    metric_name: column.quantile_values
+                    metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                    metric_value_kwargs:
+                      quantiles:
+                        - 0.05
+                        - 0.25
+                        - 0.50
+                        - 0.75
+                        - 0.95
+                  p_values:
+                    min_value: ($false_positive_threshold / 2)
+                    max_value: 1 - ($false_positive_threshold / 2)
+                - parameter_name: my_parameter_max
+                  class_name: MetricParameterBuilder
+                  metric_name: column.max
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - id: my_parameter_quantiles
+                  class_name: MetricParameterBuilder
+                  metric_name: column.quantile_values
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                  metric_value_kwargs:
+                    quantiles:
+                      - 0.05
+                      - 0.25
+                      - 0.50
+                      - 0.75
+                      - 0.95
+              configuration_builders:
+                - expectation: expect_column_min_to_be_between
+                  min_value: $my_parameter_false_positive_threshold_min.min_value
```

Comment:
It seems that we're sort of repeating ourselves here - does this information need to be specified multiple times throughout the configuration

Reply:
In reference to quantiles, etc.

Reply:
Good question. I think I am also missing some info to understand this example fully - it was pulled from James' early examples. I have added some questions to the "multi-batch Bob" user workflow (which also uses this example) https://github.com/great-expectations/great_expectations/blob/GE-28/bob-columnar-multibatch-profiler/tests/profiler/bob_user_workflow_verbose_profiler_config.yml